src/components/side-panel/coline-side-panel/coline-section.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { useAppSelector } from '../../../redux';
+import InterchangeCard from '../interchange/interchange-card';
+import { useDispatch } from 'react-redux';
+import { addColineColor, removeColineColor, updateColineColor } from '../../../redux/param/action';
+import { InterchangeInfo } from '../../../constants/constants';
+
+export default function ColineSection() {
+    const dispatch = useDispatch();
+
+    const selectedColine = useAppSelector(state => state.app.selectedColine);
+    const { coline } = useAppSelector(state => state.param);
+
+    const handleAdd = () => (info: InterchangeInfo) => {
+        dispatch(addColineColor(selectedColine, info));
+    };
+
+    const handleDelete = () => (i: number) => {
+        dispatch(removeColineColor(selectedColine, i));
+    };
+
+    const handleUpdate = () => (i: number, info: InterchangeInfo) => {
+        dispatch(updateColineColor(selectedColine, i, info));
+    };
+
+    // TODO: support mutiple coline color in one segement
```

Comment:
Should we use a different component template for this?

Reply:
If we support multiple coline color in one segment, then it should look similar like interchange-section.tsx with `<VStack>` and `map` thing.

Reply:
Then keep it as is for now